README.md

```diff
@@ -18,23 +29,32 @@ GitHub actions has recently started throwing warning messages regarding the depr
     + [Proxy Configuration](#proxy-configuration)
 - [License Summary](#license-summary)
 - [Security Disclosures](#security-disclosures)
-
 <!-- tocstop -->
 
 ## Usage
+We support four methods for fetching credentials from AWS, but we recommend that
+you use GitHub's OIDC provider in conjunction with a configured AWS IAM
+Identity Provider endpoint.
 
-Add the following step to your workflow:
+To to that, you would add the following step to your workflow:
 
 ```yaml
     - name: Configure AWS Credentials
-      uses: aws-actions/configure-aws-credentials@v1
+      uses: aws-actions/configure-aws-credentials@v2
       with:
         role-to-assume: arn:aws:iam::123456789100:role/my-github-actions-role
         aws-region: us-east-2
 ```
+This will cause the action to perform an `AssumeRoleWithWebIdentity` call and
```

Comment:
Link to docs to show exactly what is being referred to? https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRoleWithWebIdentity.html

Reply:
I don't we need to